OneCommunityGlobal/HGNRest
# Jatin make columns editable by owner

# Description
<img width="747" alt="Screenshot 2024-07-19 at 7 15 15 PM" src="https://github.com/user-attachments/assets/196f77f9-a38e-4dc7-bbb9-85adb73efdfe">

## Related PRS (if any):
To test this backend PR you need to checkout the #[2620](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2620) frontend PR.

## Main changes explained:
- Created api endpoint to update user information from user management page.

## How to test:
1. check into current branch
2. do `npm install` and `npm run build` to run this PR locally
3. Follow steps mentioned in frontend PR further.


## Screenshots or videos of changes:

https://github.com/user-attachments/assets/588efe23-0f5f-47aa-b86b-67f97c37059c
